### ISSUE TITLE
Core requirements rework

### ISSUE THREAD

**Issue by park-junha** [park-junha/PlanForGrad#61]
The database needs a separate table for the core requirements (pull request: #~~60~~#70). Once this new table is finished, it needs to be integrated into the web app. Currently, the web app pulls from a major called "Core" in the MajornEmphasis table, the requirements for which are stored as classes in the Classes table (e.g. "CTW 1" is stored in Classes and satisfies the major "Core" in the MajornEmphasis table).

**Comment by park-junha** [comment]
New pull request: #70
